lib/storageUtil.js

```diff
@@ -73,4 +75,15 @@ storageUtil.getCookieStorage = function() {
   };
 };
 
+storageUtil.testStorage = function(storage) {
```

Comment:
wonder if it's worth to check null-able of `storage`?

Reply:
I thought about that as well. Attempting to call `setItem` on `undefined` or `null` will throw a `TypeError`, which will result in returning `false`. I think we should be safe thanks to wrapping this in a `try catch`